wolfssl/ssl.h

```diff
@@ -287,6 +289,17 @@ struct WOLFSSL_GENERAL_NAME {
     } d; /* dereference */
 };
 
+struct WOLFSSL_DIST_POINT_NAME {
+    int type;
+    union {
+        WOLF_STACK_OF(WOLFSSL_GENERAL_NAME)* fullname;
+    } name;
+};
+
+struct WOLFSSL_DIST_POINT {
+    WOLFSSL_DIST_POINT_NAME* distpoint;
```

Comment:
Camel case.

Reply:
in the Python port the code is expecting the name distpoint, commit https://github.com/wolfSSL/wolfssl/pull/4351/commits/fdad39b879717000f8f45111e720be90dff96035 adds in a comment to the code